src/sysadmin/installation.md

```diff
@@ -601,6 +604,15 @@ As an example this location could be:
 
     /home/dhis/config/dhis-google-auth.json
 
+### Send an email to set up the Google Earth Engine API key
+
+If you only intend to using the key for the Google Earth Engine map layers,
```

Comment:
small grammar typo
change `If you only intend to using` to either:
- `If you only intend on using` or
- `If you only intend to use`